tools/test_runner.js

```diff
@@ -18,6 +18,8 @@ var console_wrapper = require('common_js/module/console');
 var builtin_modules =
   Object.keys(process.native_sources).concat(Object.keys(process.binding));
 
+var default_timeout = 240; // (in seconds)
```

Comment:
Yes, it seems a pretty big timeout, but a debug binary + valgrind combination can produce 100+ sec runtimes on Travis CI (moreover tomorrow I've seen a 169 sec runtime), so this value based on this corner-cases, but ofc we can adjust it. Now there is no limitation for the time of a testrun unless the test specifies a certain timeout in `testsets.json`.